src/components/shared/ManageTokensModal.tsx

```diff
@@ -82,8 +82,7 @@ export default function ManageTokensModal({
     plural: true,
   })
 
-  // eslint-disable-next-line no-extra-boolean-cast
-  const redeemDisabled = !Boolean(hasOverflow)
+  const redeemDisabled = hasOverflow ?? false
```

Comment:
@johnnyd-eth -- is this a controversial change? I saw you're the last one to make a change here. I know overflow is a core feature of JB. Just want to make sure I didn't bork anything here.